code/modules/cargo/goodies.dm

```diff
@@ -129,3 +129,9 @@
 	desc = "The simple beach ball is one of Nanotrasen's most popular products. 'Why do we make beach balls? Because we can! (TM)' - Nanotrasen"
 	cost = 200
 	contains = list(/obj/item/toy/beach_ball)
+
+/datum/supply_pack/goody/medipen_twopak
+	name = "Medipen Two-Pak"
+	desc = "Contains one standard epinephrine medipen and one standard emergency first-aid kit medipen. For when you want to prepare for the worst."
+	cost = 500
```

Comment:
my PR pegs these medipens as being worth 150 credits each, so maybe lower the price of this goody to 300 (or even 250) if my PR gets merged